io.github.monsterbirdlimited.soscap.yaml

```diff
@@ -0,0 +1,81 @@
+---
+app-id: io.github.monsterbirdlimited.soscap
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: soscap
+finish-args:
+  - --filesystem=home
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --device=dri
+  - --talk-name=org.freedesktop.FileManager1
+  - --filesystem=xdg-videos
+  - --env=G_MESSAGES_DEBUG=none
+  - --env=GST_DEBUG=3
+
+
+cleanup:
+  - '/include'
+  - '/lib/pkgconfig'
+  - '/man'
+  - '/share/doc'
+  - '/share/gtk-doc'
+  - '/share/man'
+  - '/share/pkgconfig'
+  - '*.la'
+  - '*.a'
+
+modules:
+  
+  - name: soscap
```

Comment:
this should be last since it's the "main" module.

Reply:
Thanks. Ive never done any of this before. It's my first project and experience in code so I appreciate the help.